site/src/content/docs/commands/zarf_connect.md

```diff
@@ -29,11 +29,11 @@ zarf connect { REGISTRY | LOGGING | GIT | connect-name } [flags]
 ```
       --cli-only           Disable browser auto-open
   -h, --help               help for connect
-      --local-port int     (Optional, autogenerated if not provided) Specify the local port to bind to.  E.g. local-port=42000
-      --name string        Specify the resource name.  E.g. name=unicorns or name=unicorn-pod-7448499f4d-b5bk6
-      --namespace string   Specify the namespace.  E.g. namespace=default (default "zarf")
-      --remote-port int    Specify the remote port of the resource to bind to.  E.g. remote-port=8080
-      --type string        Specify the resource type.  E.g. type=svc or type=pod (default "svc")
+      --local-port int     (Optional, autogenerated if not provided) Specify the local port to bind to.  E.g. local-port=42000. Ignored if --name is unset
+      --name string        Specify the resource name.  E.g. name=unicorns or name=unicorn-pod-7448499f4d-b5bk6. Ignored if --name is unset
```

Comment:
Probably an oversight, but the `Ignored if --name is unset` is part of the `--name` flag description but should not be